From 4a91771fb95b38d356086c0a3d7bd7be6edea7e0 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Mon, 12 Aug 2024 05:53:13 -0600
Subject: [PATCH] Bump Security String to 2024-08-05

Implemented:
============
CVE:            References:    Type:  Severity:  Updated AOSP versions:
CVE-2023-21351  A-232798676    EoP    High       12, 12L, 13
CVE-2024-34727  A-287184435    ID     High       12, 12L, 13, 14
CVE-2024-34735  A-336490997    EoP    High       12, 12L, 13
CVE-2024-34736  A-288549440    ID     High       12, 12L, 13, 14
CVE-2024-34739  A-294105066    EoP    High       12, 12L, 13, 14
CVE-2024-34741  A-318683640    EoP    High       12, 12L, 13, 14

Previously Implemented:
=======================
CVE:            References:    Type:  Severity:  Updated AOSP versions:  Prior Change:
CVE-2023-20971  A-225880325    EoP    High       12, 12L, 13, 14         4b0cceb62625

Not Implemented:
================
None

Not Applicable (platform source):
=================================
CVE:            References:    Type:  Severity:  Updated AOSP versions:
CVE-2024-34731  A-319210610    EoP    High       12, 12L, 13, 14
CVE-2024-34734  A-304772709    EoP    High       13, 14
CVE-2024-34737  A-283103220    EoP    High       12, 12L, 13, 14
CVE-2024-34738  A-336323279    EoP    High       13, 14
CVE-2024-34740  A-307288067    EoP    High       12, 12L, 13, 14
CVE-2024-34742  A-335232744    DoS    High       14
CVE-2024-34743  A-336648613    EoP    High       14

Change-Id: I7bd13b4b5684d5e24ecaec8f5a55afb728280565
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 58b3d1294a..7c55cf9f7a 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -250,7 +250,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-    PLATFORM_SECURITY_PATCH := 2024-07-05
+    PLATFORM_SECURITY_PATCH := 2024-08-05
 endif
 .KATI_READONLY := PLATFORM_SECURITY_PATCH
 
-- 
2.40.1

